internal/manifests/collector/configmap_test.go

```diff
@@ -17,6 +17,8 @@ package collector
 import (
 	"testing"
 
+	"github.com/open-telemetry/opentelemetry-operator/internal/manifests/manifestutils"
+	"github.com/open-telemetry/opentelemetry-operator/internal/naming"
```

Comment:
these imports should be moved to a separate group

Reply:
We should update the gci config to enforce this.